lib/travis/api/v3/models/cron.rb

```diff
@@ -0,0 +1,112 @@
+module Travis::API::V3
+  class Models::Cron < Model
+
+    belongs_to :branch
+
+    def start
```

Comment:
This should probably move to a query object. Models should not have knowledge about the underlying DB structure or Sidekiq (we pretend that AR doesn't follow the Active Record pattern).